FlorenceBML/BMLacq776.xml

```diff
@@ -30,10 +31,731 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                        <title ref="LIT1295Dersan#HomilyHamle"/> as found in <locus target="#ms_i1.9.1"/> in this record. </note>
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.2">
+                                    <locus from="35rb" to="35vb"/>
+                                    <title ref="LIT1295Dersan#CommemorativeTerr">When Mikāʾel was sent to Yāʿqob.</title>
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.3">
+                                    <locus target="#35vb"/>
+                                    <title ref="LIT1295Dersan#SalamTerr">Salām for the month Ṭərr</title>
+                                </msItem>
+                                <msItem xml:id="ms_i1.3.4">
+                                    <locus from="35vb" to="36va"/>
+                                    <title ref="LIT1295Dersan#MiracleTerr">Miracle of the rich woman</title>
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="ms_i1.4">
+                                <title ref="LIT1295Dersan#Yakkatit"/>
+                                <msItem xml:id="ms_i1.4.1">
+                                    <locus from="36vb" to="42ra"/>
+                                    <title ref="LIT1295Dersan#HomilyYakkatit">Homily on a poor man of great faith</title>
+                                    <incipit xml:lang="gez"><hi rend="rubric">በ<gap reason="lost" unit="chars" quantity="1"/> በስመአብ፡ ወወልድ፡ ወመንፈስ፡ ቅ፡</hi> 
+                                        ፩አምላክ፡ ድርሳን፡ ዘቀድሰ፡ <hi rend="rubric">ሚካኤል፡</hi> ሊቀ፡ መላእክት፡ ዘይትነበብ፡ አመ፲ወ፪፡ 
+                                        ለ<subst><del>ጥር</del><add rend="overstrike">የካ፡</add></subst> ቲት፡ በሰላመ፡ እ<hi rend="ligature">ግዚ</hi>አብሔር፡ 
+                                        ዓሜን፡ ወኮነ፡ ፩ዱ፡ ነዳይ፡ ውስተ፡ ፩ብሔር፡ ወዓቢይ፡ ሃይማኖቱ፡ ይጼሊ፡ መአልተ፡ ወሌሊተ፡ ወይብል፡ ኦ<gap reason="omitted" unit="chars" quantity="4"/> 
+                                        ሊቅየ፡ ወሊቆሙ፡ ለመላእክት፡ ተሣሃለኒ፡ እስመ፡ ሐካይ፡ ብእሲሁ፡ ወኢይወጽእ፡ እምነ፡ ቤቱ፡ ወአልቦቱ፡ ተግባር፡ ወኢስራኅ፡ ወኢየአምር፡ ንጊደ፡ ወእምዝ፡ ወእምዝ፡ 
+                                        መጽአ፡ ቅዱስ፡ <hi rend="rubric">ሚካኤል፡</hi> ወአስተርአዮ፡ በኅልም፡ ወይቤሎ፡ አንተ፡ ብእሲ፡ ሀካይ፡ ወኢትትገበር፡ አሐደ፡ እምግብርናት፡ ወትብለኒ፡ 
+                                        መአልተ፡ ወሌሊተ፡ ተሣሃለኒ፡ አይነ፡ </incipit>
+                                    <explicit xml:lang="gez">ወወሀቦ፡ ለቤተ፡ ክርስቲያን፡ ወአግበረ፡ ፃሕለ፡ ዘቅዱስ፡ <hi rend="rubric">ሚካኤል፡</hi> ወነዳይኒ፡ ነሥአ፡ ሃምሳ፡ 
+                                        ረ<pb n="42r"/>በሁ፡ አግበረ፡ ጽዋዓ፡ ለቤተ፡ ክርስቲያን፡ ዘቅዱስ፡ <hi rend="rubric">ሚኤል፡</hi> ወተዓረኩ፡ ፪ሆሙ፡ አምዓሜሃ፡ እለት፡ ዕሰሙ፡ ወፍቅሩ፡ 
+                                        የሀሉ፡ ምስለ፡ ፍቁሩ፡ <gap reason="omitted" unit="chars" quantity="4"/> ለአለመ፡ ዓሚ፡ ከመ፡ </explicit>
+                                    <note>Anonymous here, but elsewhere as in <ref target="#">A 84 = N 137</ref> attributed to "Severus, patriarch of Alexandria"
```

Comment:
As is clear from the list of abbreviations A stands for d'Abbadie and N for Conti Rossini Notice, so the reference is to BNFabb84 https://archivesetmanuscrits.bnf.fr/ark:/12148/cc103059f